Create ui-c8y-1019-11-0-the-size-of-the-measurement-displaying-line-was-increased.md (#1820)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1019-11-0-the-size-of-the-measurement-displaying-line-was-increased.md

---
date: ""

> title: Increase measurement display line size

Reviewer: ```suggestion
title: Improved measurement display in Linear Gauge and Silo widgets
```

> title: The size of the measurement displaying line was increased

Author: The Claude AI suggested to use the following title:
```suggestion
title: Increase measurement display line size
```

product_area: Application enablement & solutions
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component-YdSEScrEC
    label: Cockpit
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-52889
version: 1019.11.0
---

> The size of the measurement displaying line was increased

Author: The Claude AI suggested to use the following body:
```suggestion
Increased the height of the line used to display measurements to improve readability. This makes measurements easier to read by providing more vertical space for the text.
```

> The height of the line used to display measurements has been increased. This change improves readability by providing more vertical space for the text.

Reviewer: ```suggestion
The measurement display line size has been increased in both the "Linear Gauge" and the "Silo" widget. This enhancement improves readability, making it easier for users to interpret measurements at a glance.
```
